### DISCUSSION TITLE
Year and Month selection

### DISCUSSION THREAD

**Discussion by AnshulVats55** [shadcn-ui/ui#4937] [Q&A]
Hi team,

I was wondering how can I jump to a particular Month or Year when using your Date Picker or Calendar components.
Though it provides "Prev" and "Next" arrows to change the month, but what If I want to select the Month and Year, directly without pressing these "Prev" and "Next" arrows a thousand times 😒
Is there anything I'm missing while using your Date Picker/Calendar?

Looking forward to your response.

Regards,
Anshul Vats

**Comment by victornwakpa** [comment]
Check this out:
https://gist.github.com/mjbalcueva/1fbcb1be9ef68a82c14d778b686a04fa

**Reply by AnshulVats55** [reply]
Thank you brother 💕
